core/src/main/java/org/kiwix/kiwixmobile/core/JNIInitialiser.kt

```diff
@@ -50,7 +50,7 @@ internal class JNIInitialiser @Inject constructor(context: Context, jniKiwix: JN
       icuDir.absolutePath
     } catch (e: Exception) {
       Log.w(TAG_KIWIX, "Error copying icu data file", e)
-      // TODO: Consider surfacing to user
+      // TODO Consider surfacing to user
```

Comment:
A good approach to addressing the styling issue would not affect the TODO block, as it still appears in the IDE's TODO list, indicating pending tasks :+1: :+1: .

![Screenshot from 2023-05-22 16-04-00](https://github.com/kiwix/kiwix-android/assets/34593983/5522b79b-8d2a-47f1-af5e-d0efd1d002c6)

Reply:
It does appear in the TODO list of the IDE , and other TODO's are also written the same way .should i let it be?

Reply:
@shashank-0-0 It seems there was a misunderstanding. What I meant to say is that your approach is a good solution for resolving the styling issue, and it won't have any impact on the existing TODO list.

Reply:
My bad ,i thought i had to change the approach . And what about [this](https://github.com/kiwix/kiwix-android/pull/3323#discussion_r1200650739)